core/spec/models/spree/promotion_handler/coupon_spec_with_gift_card.rb

```
@@ -0,0 +1,68 @@
+require 'spec_helper'
```

RuboCop (1.73)

[convention] Spec path should end with `_spec.rb`.

(RSpec/SpecFilePathSuffix)